ofrak_core/ofrak/core/ihex.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import re
+import sys
+from dataclasses import dataclass
+from typing import List, Union, Tuple
+
+from bincopy import BinFile  # type:ignore
```

Comment:
Out of curiosity, why does this need a type ignore? I haven't noticed imports causing mypy errors before.

Reply:
It was to deal with the mypy error about not being able to type check it:
> ofrak/core/ihex.py:7: error: Skipping analyzing "bincopy": module is installed, but missing library stubs or py.typed marker

The way we usually deal with them is adding an ignore for that module in the `mypy.ini` file. Updated.